src/hotspots-plugin.tsx

```diff
@@ -105,6 +112,14 @@ export class HotspotsPlugin extends KalturaPlayer.core.BasePlugin {
     };
   }
 
+  private _calculateStyle(cuepoint: LayoutHotspot, scaleCalculation: ScaleCalculation): Style {
+    const {rawLayout, styles} = cuepoint;
+    return {
+      fontSize: parseInt(styles['font-size'])/ rawLayout.stageWidth * scaleCalculation.width,
```

Comment:
are these values always set ?

Reply:
yes, kea sent the data like this

Reply:
what about kms ? it has a different editor i think